DataTransfer/TransferFiles.py

```diff
@@ -0,0 +1,11 @@
+#python 3.6
+from github import Github
+token = "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"
+git_user = Github(token)
+repo = git_user.get_user().get_repo("TeamProjectsIIData")
+folder_prefix = "Data/"
+test_file = folder_prefix + "TestData.txt"
+contents = repo.get_contents("")
```

Comment:
What is this line doing? It might be necessary but "contents" isn't being used. "content" also isn't being used below.

Reply:
removed unnecessary code